examples/parameter_replacement/parameter_replacement.ipynb

```diff
@@ -0,0 +1,316 @@
+    "            print(f\"        {k}: {v}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Updating Parsley with Sage angles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-danger\" style=\"max-width: 700px; margin-left: auto; margin-right: auto;\">\n",
+    "    <b>🛑 For Educational Purposes Only</b><br />\n",
+    "    Don't combine parameters from different force fields or \"update\" one part of a force field without careful consideration. Parameters are designed and optimized to work within a particular context, and moving them to a completely different force field is usually a bad idea.\n",
+    "</div>\n",
+    "\n",
+    "Just to demonstrate changing parameters, lets \"update\" our Parsley Interchange to use Sage angle parameters. This will definitely hurt force field performance, as each set of angle parameters are optimized for different non-bonded parameters, but it demonstrates the API, and we won't be running any simulations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Generally, updating a `PotentialHandler` involves updating both dictionaries, `slot_map` and `potentials`. Since both of our Interchanges use the same SMIRKS codes and parameter types, we could skip updating the `slot_map`, but it's best practice to do both steps every time. It's good practice to clear the old values so that you don't accidentally wind up with redundant parameters (unless that's what you want), so let's start with that:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pars_sys.handlers[\"Angles\"].slot_map.clear()\n",
+    "pars_sys.handlers[\"Angles\"].potentials.clear()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The [`store_matches`] method updates `slot_map`. It takes a topology and some object that describes the parameters (for SMIRNOFF, a `ParameterHandler`) and applies the described force field, storing the appropriate atom indices in the calling handler. If there are clashes, the new values replace the old.\n",
+    "\n",
+    "[`store_matches`]: https://openff-interchange.readthedocs.io/en/stable/_autosummary/openff.interchange.components.potentials.PotentialHandler.html#openff.interchange.components.potentials.PotentialHandler.store_matches\n",
+    "[`store_potentials`]: https://openff-interchange.readthedocs.io/en/stable/_autosummary/openff.interchange.components.potentials.PotentialHandler.html#openff.interchange.components.potentials.PotentialHandler.store_potentials"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pars_sys.handlers[\"Angles\"].store_matches(sage_ff[\"Angles\"], topology=pars_sys.topology)"
```

Comment:
This line was previously:

```python
sys_100.handlers[\"Angles\"].store_matches(parsley_100["Angles"], topology=top)
```

I changed it to

```python
sys_100.handlers[\"Angles\"].store_matches(parsley_130["Angles"], topology=sys_100.topology)
```

because I think it was an error, `store_matches` takes the new parameter handler as input, not the old one. Can you confirm this was an error?